classes.js

```diff
@@ -287,8 +287,8 @@ Discord.Structures.extend("Guild", G => {
 						this.members.add(member);
 					}
 				}
-				if(!this.members.cache.has(this.client.user.id)) {
-					this.members.fetch(this.client.user.id).catch(() => {});
+				if(!this.members.cache.has(this.client.user?.id)) {
```

Comment:
this is a problem: optional chaining short circuits to `undefined` so if client.user is falsey, then its doing `.has(undefined)` which will count as user not cached and attempt to `fetch(undefined)`